dart/generate_map/README.md

```diff
@@ -0,0 +1,72 @@
+# 🗺️ Generate Map
+
+A Dart Cloud Function that generates a static map base64 encoded image at given latitude and longitute.
+
+_Example input 1:_
+
+```json
+{
+    "lat": "50", 
+    "lng": "50"
+}
+```
+
+_Example output 1:_
+
+
+```json
+{
+    "success": "true",
+    "image": "9j..longString.dasdsdasd/sadas/d/asd/asdasfasasfssdafad="
+}
+```
+_Example input 2:_
+
+```json
+{
+    "lat": "50", 
+    "lng": "-270"
+}
+```
+
+_Example output 2:_
+
+
+```json
+{
+    "success": "false",
+    "message": "The longitude must be between -180 and 180"
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Let's remove references to "environment" since it's no longer called environment variables:

```suggestion
## 📝 Variables

List of variables used by this cloud function:
```

Reply:
Will do, but then how do I access them in the function?

Reply:
`req.variables` instead of `req.env`.